extensions/Dockerfile

```diff
@@ -24,6 +24,8 @@ RUN if [[ -n "${VARIANT}" ]]; then MANIFEST="manifest-${VARIANT}.yaml"; EXTENSIO
 ## current p8/s390x. See https://github.com/openshift/os/issues/1000
 FROM quay.io/fedora/fedora:40 as builder
 COPY --from=os /usr/share/rpm-ostree/extensions/ /usr/share/rpm-ostree/extensions/
+RUN rm -f /etc/yum.repos.d/*.repo \
+&& curl -L https://raw.githubusercontent.com/coreos/fedora-coreos-config/rhcos-4.17/fedora.repo -o /etc/yum.repos.d/fedora.repo
```

Comment:
@marmijo can we change the repo to testing-devel here?

Reply:
Most of the tests have passed and all approvals are in on this PR (and we're going into a long weekend). I'd like to land it today to test building 4.17 in the ITUP cluster. 
Also, the fedora.repo file in `rhcos-4.17` and `testing-devel` are identical, so it would have no real impact to do it right now in this PR.

Reply:
sounds good